Infrustructure/Integration/SeaRoutesClient.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Domain.Models;
+using RestSharp;
+using RestSharp.Serialization.Json;
+
+namespace Infrustructure.Integration
+{
+    public interface ISeaRoutesClient
+    {
+        Task<SeaRoutesResult> GetClosestPort(SeaRoutesRequest requestParams);
```

Comment:
Naming convention for async Tasks is  ending function name with  Async in the end. 
e.g;  `Task<SeaRoutesResult> GetClosestPortAsync(SeaRoutesRequest requestParams);`